src/main/kotlin/me/owdding/lib/overlays/Position.kt

```diff
@@ -0,0 +1,28 @@
+package me.owdding.lib.overlays
+
+import com.teamresourceful.resourcefulconfigkt.api.ObjectKt
+import tech.thatgravyboat.skyblockapi.helpers.McClient
+
+class Position(
+    x: Int = 0,
+    y: Int = 0,
+    scale: Float = 1.0f,
+) : ObjectKt() {
```

Comment:
I don't really like the dependency on rconfig in here we should make this an interface and have a rconfig specific version like ConfigPosition